lib/handler.ex

```diff
@@ -70,6 +85,7 @@ defmodule SAXMap.Handler do
     list_to_map(rest, prepared)
   end
 
+  @compile {:inline, put_or_concat_to_map: 3}
```

Comment:
```suggestion
  @compile {:inline, put_or_concat_to_map: 4}
```
The input number of arguments is 4.